packages/smooth_app/lib/pages/product/common/smooth_chip.dart

```diff
@@ -20,20 +19,8 @@ class SmoothChip extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final ColorScheme colorScheme = Theme.of(context).colorScheme;
-    final Color? foregroundColor = materialColor == null
-        ? null
-        : SmoothTheme.getColor(
-            colorScheme,
-            materialColor!,
-            ColorDestination.BUTTON_FOREGROUND,
-          );
-    final Color? backgroundColor = materialColor == null
-        ? null
-        : SmoothTheme.getColor(
-            colorScheme,
-            materialColor!,
-            ColorDestination.BUTTON_BACKGROUND,
-          );
+    final Color foregroundColor = colorScheme.onPrimaryContainer;
+    final Color backgroundColor = colorScheme.primaryContainer;
```

Comment:
I don't know `primaryContainer` and `onPrimaryContainer`, but my hunch tells me it `onPrimaryContainer` should be the background color, not the other way around.
Wouldn't background=secondary / foreground=onSecondary make sense here?